docs/builtins/Time/parse-time.md

```diff
@@ -1,32 +1,60 @@
 ## parse-time
-Use `parse-time` to construct time from a UTC value using a specified format. See ["Time Formats" docs](https://docs.kadena.io/reference/functions/time#time-formatsh-299815639) for supported formats.
+
+Use `parse-time` to construct a UTC formatted time string from an input time that uses the specified format. 
+The following table provides a summary of common time formatting codes:
+
+| Format | Purpose |
```

Comment:
Personally I think it is fine to repeat the definition in each of the two functions.